src/slash-bedrock/libexec/brl-which

```diff
@@ -67,6 +68,19 @@ which_bin() {
 	done
 }
 
+which_bin_all() (
+	min_args "${#}" "1"
+
+	while [ -n "${1:-}" ]; do
+		for stratum in $(list_strata); do
+			if path="$(strat -r "${stratum}" /bedrock/libexec/busybox which "$1" 2>/dev/null)"; then
```

Comment:
I can add handling for "file/binary not found" both here and in `which_file_all`, but what should be the behavior? Something like this?

```
$ brl which -a git 0ad cmake
arch: git
WARNING: 0ad not found in any strata
void: cmake
# exit with status 1
```

Reply:
I don't think it makes much sense to phrase it as a warning, as it's really just the absence of something. Perhaps something closer to just `0ad not found in any strata`?

Reply:
That would probably depend on the exact format; see [above](https://github.com/bedrocklinux/bedrocklinux-userland/pull/212/files/0ac59f6294003c1f6c0342a132e81259b1cd6218#r567808165). e.g. for some it could be `0ad not found`; for others `0ad: not found` might make more sense. Colorization might help too.